content/administration/maintain/google_oauth.rst

```diff
@@ -0,0 +1,240 @@
+:guilabel:`Outgoing Email Servers` to configure the external Gmail account.
+
+.. image:: google_oauth/outgoing-servers.png
+   :class: img-fluid
+   :align: center
+   :alt: Configure Outgoing Email Servers in Odoo.
+
+Then, :guilabel:`create` a new email server and select the option for :guilabel:`Gmail`. Next, fill
+in the :guilabel:`Description` (can be anything) and the email :guilabel:`Username` and click on
+:guilabel:`Connect your Gmail account`.
+
+A new window from :guilabel:`Google` appears to complete the authorization process. Select the
+appropriate email address that is being configured in Odoo.
+
+If the email address is a personal account, an extra step pops up, click :guilabel:`Continue` to
+allow the verification and connect the Gmail account to Odoo.
+
+.. image:: google_oauth/google-unverified.png
+   :class: img-fluid
+   :align: center
+   :alt: Google Hasn't Verified This App.
+
+Finally, allow Odoo to access the Google account by clicking on :guilabel:`Continue` or
+:guilabel:`Allow`.
+
+After that, the page navigates back to the newly configured outgoing email server in Odoo. The
+configuration automatically loads the token in Odoo, and a graphic stating
+:guilabel:`Gmail Token Valid` appears in green.
+
+.. image:: google_oauth/green-token.png
+   :class: img-fluid
+   :align: center
+   :alt: Configure Outgoing Email Servers in Odoo.
+
+Finally, save the settings and :guilabel:`Test the Connection`.
+
+The Odoo database can now send safe, secure emails through Google using OAuth authentication.
+
+Google OAuth FAQ
+================
+
+Production VS Testing Publishing Status
+---------------------------------------
+
+Choosing :guilabel:`Production` as the :guilabel:`Publishing Status` (instead of Testing) will
+display the following warning message:
+
+.. image:: google_oauth/published-status.png
+   :class: img-fluid
+   :align: center
+   :alt: OAuth is Limited to 100 Sensitive Scope Logins.
+
+To correct this warning, navigate to the 
+`Google APIs platform <https://console.cloud.google.com/apis/credentials/consent>`__. If the app is
+in :guilabel:`Production`, then click :guilabel:`Back to Testing` to correct the issue.
+
+.. image:: google_oauth/back-to-testing.png
+   :class: img-fluid
+   :align: center
+   :alt: Back to Testing in Google OAuth.
+
+No Test Users Added
+-------------------
+
+If no test users are added to the OAuth consent screen, then a 403 access denied error will
+populate.
+
+.. image:: google_oauth/403-error.png
+   :class: img-fluid
+   :align: center
+   :alt: 403 Access Denied Error.
+
+To correct this error, return to the :guilabel:`OAuth consent screen` under
+:guilabel:`APIs & Services` and add test user(s) to the app. The email added should be the one
+being configured to send in Odoo.
+
+Gmail Module not Updated
+------------------------
+
+Should the :guilabel:`Gmail Module` in Odoo not be updated to the latest version, then a Forbidden
```

Comment:
* replace `Should` with `If`
* add `has` after `in Odoo`
* add a `:guilabel:` to `Forbidden`